votor/src/consensus_pool.rs

```diff
@@ -213,28 +211,24 @@ impl ConsensusPool {
             if accumulated_stake as f64 / (total_stake as f64) < limit {
                 continue;
             }
-            let mut vote_certificate_builder = VoteCertificateBuilder::new(cert_id);
+            let mut cert_builder = VoteCertificateBuilder::new(cert_type);
             vote_types.iter().for_each(|vote_type| {
                 if let Some(vote_pool) = self.vote_pools.get(&(slot, *vote_type)) {
                     match vote_pool {
                         VotePool::SimpleVotePool(pool) => {
-                            vote_certificate_builder.aggregate(pool.votes()).unwrap();
+                            cert_builder.aggregate(pool.votes()).unwrap();
                         }
                         VotePool::DuplicateBlockVotePool(pool) => {
-                            if let Some(votes) = pool.votes(block_id.as_ref().expect(
-                                "Duplicate block pool for {vote_type:?} expects a block id for \
-                                 certificate {cert_id:?}",
```

Comment:
Why are we removing the error message here?

Reply:
ah this is a spillover from a future PR where this expect goes away.  Added back in.